### PR TITLE
docs: add datanode client options & procedure options

### LINKED ISSUE #481: Update docs for feat: meta procedure options
A document change request is generated from
https://github.com/GreptimeTeam/greptimedb/pull/1937

### PR DESCRIPTION
**Pull request by WenyXu**
## What's Changed in this PR

close #481 #488
add datanode client options & procedure options

## Checklist

- [ ] Please confirm that all corresponding versions of the documents have been revised.
- [ ] Please ensure that the content in `summary.yml` matches the current document structure when you changed the document structure.
- [ ] This change requires follow-up update in localized docs.

### CODE REVIEW

**Review by fengjiachun — APPROVED** [review]
LGTM

File: docs/nightly/zh/user-guide/operations/configuration.md [hunk 1]
```diff
@@ -496,7 +492,7 @@ mode = "standalone"
 
 ### 仅限于 Metasrv 的配置
 
-```toml
+````toml
```
Comment (killme2008) [inline docs/nightly/zh/user-guide/operations/configuration.md:495]: Why needs an extra quote?

Comment (nicecui) [inline docs/nightly/zh/user-guide/operations/configuration.md:495]:
```suggestion
```toml
```

**Review by nicecui — COMMENTED** [review]
please copy the content to v0.9

**Review by nicecui — APPROVED** [review]

**Pull request merged**